dev-libs/gobject-introspection/patches/gobject_introspection-1.56.0.patchset

```diff
@@ -0,0 +1,54 @@
+From 5ff93016bdba5e945107f406657a6533c36daae3 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 12 Aug 2017 20:13:53 +0200
+Subject: use readelf instead of ldd
+
+
+diff --git a/giscanner/shlibs.py b/giscanner/shlibs.py
+index 525bdba..ef560a8 100644
+--- a/giscanner/shlibs.py
++++ b/giscanner/shlibs.py
+@@ -108,7 +108,7 @@ def _resolve_non_libtool(options, binary, libraries):
+         if platform_system == 'Darwin':
+             args.extend(['otool', '-L', binary.args[0]])
+         else:
+-            args.extend(['ldd', binary.args[0]])
++            args.extend(['readelf', '-d', binary.args[0]])
+         proc = subprocess.Popen(args, stdout=subprocess.PIPE)
+         patterns = {}
+         for library in libraries:
+@@ -126,7 +126,7 @@ def _resolve_non_libtool(options, binary, libraries):
+                 m = pattern.search(line)
+                 if m:
+                     del patterns[library]
+-                    shlibs.append(m.group(1))
++                    shlibs.append(m.group(1)[0:-1])
+                     break
+ 
+         if len(patterns) > 0:
+-- 
+2.16.2
+
+
+From a9e29abee77baf9facfd3c6ce57d3feff2b5ec39 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 17 Mar 2018 20:10:13 +0100
+Subject: Use current glib-2.0 version
+
+
+diff --git a/configure.ac b/configure.ac
+index 367f2db..32fbf6c 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -128,7 +128,7 @@ GIR_DIR="$EXPANDED_DATADIR/$GIR_SUFFIX"
+ AC_SUBST(GIR_DIR)
+ AC_DEFINE_UNQUOTED(GIR_DIR, "$GIR_DIR", [Director prefix for gir installation])
+ 
+-PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.56.0])
```

Comment:
```diff
--PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.56.0])
+-PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.56.1])
 +PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.53.0])
```

Reply:
Strange, I thought I changed it as we are not yet at 2.56.1 ...

Reply:
2.56.1 = related to the build you tried for 1.56.1 ...

Reply:
Oops, sorry, I forgot to mention that the changes I suggested in this review are for gobject-introspection `1.56.1`.

Reply:
though so ;) bumped version :)